src/main/java/at/hannibal2/skyhanni/features/nether/reputationhelper/dailykuudra/DailyKuudraBossHelper.kt

```diff
@@ -45,11 +44,11 @@ class DailyKuudraBossHelper(private val reputationHelper: CrimsonIsleReputationH
         if (!SkyHanniMod.feature.misc.crimsonIsleReputationHelper) return
 
         val message = event.message
-        if (message != "                               §r§6§lKUUDRA DOWN!") return
+        if (!message.contains("KUUDRA DOWN!") || message.contains(":")) return
```

Comment:
Are we sure there is no other chat message with the words kuudra down in it?

Reply:
It doesn't seem to be the case, I added the semicolon check so even if people write the message in chat it doesn't work. I did this change because the old way wasn't working